frontend/src/routes/Login.tsx

```diff
@@ -65,13 +66,12 @@ const Login: React.FC<Props> = ({ queryRef }) => {
         // Don't render anything when a redirect is triggered.
         ? null
         : <Outer>
+            <Header loginMode />
             <div css={{
-                height: "calc(1.7 * var(--header-height))",
-                maxHeight: "35vh",
-                padding: `${HEADER_BASE_PADDING}px 0`,
-                textAlign: "center",
-            }}><Logo /></div>
-
+                margin: 0,
+                height: 2,
+                backgroundColor: "var(--grey92)",
+            }}/>
```

Comment:
Maybe this "border below header" should be factored out so that this doesnt duplicate code from `Root.tsx`?